rdwatch/scoring/views/site_observation.py

```diff
@@ -35,13 +35,21 @@
 
 class GenerateImagesSchema(Schema):
     constellation: list[Literal['WV', 'S2', 'L8', 'PL']] = ['WV']
+    worldviewSource: Literal['cog', 'nitf'] | None = None
     dayRange: int = 14
     noData: int = 50
     overrideDates: None | list[str] = None
     force: bool = False
     scale: Literal['default', 'bits', 'custom'] = 'bits'
     scaleNum: None | list[int] = None
     bboxScale: None | float = 1.2
+    pointArea: None | float = 200
+
+    @root_validator
+    def validate_worldview_source(cls, values: dict[str, Any]):
+        if 'WV' in values['constellation'] and values['worldviewSource'] is None:
+            raise ValueError('worldviewSource is required for WV constellation')
+        return values
```

Comment:
Didn't we remove this because it was causing problems? I don't recall what the exact issue was, though.

Reply:
https://github.com/ResonantGeoData/RD-WATCH/pull/472#discussion_r1709952201

So there was a default version located above.
Then I modified it in the STAC Query PR. 
https://github.com/ResonantGeoData/RD-WATCH/pull/493/files#diff-77c78040069f3371d705c88e756aefd249aa57d2b0d893c699d8d6ab58faad63

The update allows you provide a worldview-source (NITF/COG) even if you don't have WV selected.  So that way we can have a global default value.  I'm using the same one as the core version.